package-lock.json

```diff
@@ -0,0 +1,7111 @@
+{
```

Comment:
Because I've decided to use `yarn` I don't want to have package-lock files in the repo.
It's confusing and unclear which lockfile is the main source of truth.